client.go

```diff
@@ -37,6 +38,10 @@ func NewClient(url, username, password string) (c *Client) {
 
 	c.httpClient = &http.Client{}
 
+	if len(useBetaApi) > 0 {
```

Comment:
Do you have a test case as an example on how to use this?

Reply:
Ah yes, it is not a test-case actually, I was testing in the main but I didn't pushed it. I will push a test-case soon.

Reply:
I updated this test case to validate the default value (false) and the new value with parameter (true) https://github.com/renzors/testrail/blob/8ba6c275dd2fd0e8bddd7e15f776ba6176e22c45/client_test.go#L48